trendico/tasks.py

```diff
@@ -0,0 +1,42 @@
+from email.mime.text import MIMEText
+from django.conf import settings
+from django.contrib.auth.models import User
+from trendico.models import Order
+from celery import shared_task
+import smtplib
+
+
+@shared_task
+def send_order_notification_email(order_id):
+    try:
+        order = Order.objects.get(pk=order_id)
+        admins = User.objects.filter(is_staff=True, is_superuser=True)
+        admin_emails = admins.values_list('email', flat=True)
```

Comment:
can be one liner code:
`User.objects.filter(is_staff=True, is_superuser=True).values_list('email', flat=True)`